app/graphql/types/organizations/update_organization_input.rb

```diff
@@ -24,6 +24,9 @@ class UpdateOrganizationInput < BaseInputObject
 
       argument :timezone, Types::TimezoneEnum, required: false
 
+      argument :document_numbering, Types::Organizations::DocumentNumberingEnum, required: false
+      argument :document_number_prefix, String, required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `document_number_prefix` should appear before `document_numbering`.